src/fontcache_internal.h

```diff
@@ -19,59 +19,4 @@ static const int MAX_FONT_SIZE = 72; ///< Maximum font size.
 static const byte FACE_COLOUR = 1;
 static const byte SHADOW_COLOUR = 2;
```

Comment:
fontcache_internal is now left with just 3 constants. Can they go somewhere else?

Reply:
I think so, They are all only used by TrueTypeFontCache implementations, but the two colour definitions could be considered generic enough to just live in fontcache.h.

I think for now I should simply move them along with the other TrueType bits.